models/_generate_layers.py

```diff
@@ -496,6 +497,50 @@ def collect_layers():
   return ls
 
 
+def get_super_call_params(layer: LayerBase) -> str:
+  """
+  Inspects the given layer in RETURNN, extracts the super call from there, removes unwanted parameters
+  and then returns a string that is ready to be used in the super call for the _generated_layers file.
```

Comment:
What is being returned should be documented under `:return:` and not in the generic description.

There should be an empty line after the generic description.

Reply:
Updated the documentation, is it more clear now?